docs/quick-start.md

```diff
@@ -408,4 +409,12 @@ kubectl delete -l "kubewarden" mutatingwebhookconfigurations.admissionregistrati
 
 ## What's next?
 
-Now, you are ready to deploy Kubewarden! Have a look at the policies on [artifacthub.io](https://artifacthub.io/packages/search?kind=13), on [GitHub](https://github.com/topics/kubewarden-policy), or reuse existing Rego policies as shown in the [following chapters](tutorials/writing-policies/rego/01-intro-rego.md).
+Now, you are ready to deploy Kubewarden! Have a look at the policies on
+[artifacthub.io](https://artifacthub.io/packages/search?kind=13), on
+[GitHub](https://github.com/topics/kubewarden-policy), or reuse existing Rego
+policies as shown in the [following
+chapters](tutorials/writing-policies/rego/01-intro-rego.md). Here is the list of
+available policies on ArtifactHub:
```

Comment:
```suggestion
chapters](tutorials/writing-policies/rego/01-intro-rego.md).

Finally, here is the list of available policies on ArtifactHub.
```

Reply:
I think the @jhkrug approach a better way to list the policies and makes this suggestion obsolete. Therefore, I'll skip this change. Ok? ;)